SofaKernel/modules/SofaBaseTopology/GridTopology.h

```diff
@@ -184,8 +184,9 @@ using MeshTopology::getHexahedron;
     /// Get Cube index, similar to \sa hexa method
     int cube(int x, int y, int z) const { return hexa(x,y,z); }
 
-	/// Get the actual dimension of this grid using Enum @sa Grid_dimension
-	Grid_dimension getDimensions() const;
+    /// Get the actual dimension of this grid using Enum @sa Grid_dimension
+    Grid_dimension getDimensions() const;
```

Comment:
don't have "the eyes in front of the holes".. is there a change here?

Reply:
Nope, only formatting.